plugins/builds/index.js

```diff
@@ -675,10 +675,10 @@ async function createJoinObject(nextJobs, current, eventFactory) {
 
             if (externalEvent) {
                 pipelineObj.event = externalEvent;
-                jobs = workflowParser.getSrcForJoin(externalEvent.workflowGraph, { jobName: jName });
+                jobs = workflowParser.getSrcForJoin(event.workflowGraph, { jobName: jName });
```

Comment:
why aren't we using `externalEvent` here?

Reply:
Good catch. Fixed it.